features/product/managing_products/adding_products_with_position_and_taxon.feature

```diff
@@ -0,0 +1,113 @@
+@managing_products
+Feature: Adding a new product
```

Comment:
This feature title  and body do not describe well enough what we are checking here.

```
Feature: Adding a new product with a position within a taxon
In order to organize products within taxons in my shop
As an Administrator
I want to assign positions in taxons to products when I create them
```

I would suggest sth like that.